src/components/layers/toolbar/LayerToolbar.js

```diff
@@ -35,10 +35,10 @@ const LayerToolbar = ({
             <IconButton
                 tooltip={!hasError ? i18n.t('Toggle visibility') : null}
                 onClick={toggleLayerVisibility}
-                className={{
+                className={cx(styles.visibilityButton, {
```

Comment:
Is this style defined already or is it new?

Reply:
Right, `visibilityButton` isn't needed.